polytope_server/common/request_store/request_store.py

```diff
@@ -68,7 +68,7 @@ def collect_metric_info(
         """Collect dictionary of metrics"""
 
 
-type_to_class_map = {"mongodb": "MongoRequestStore"}
+type_to_class_map = {"mongodb": "MongoRequestStore", "dynamodb": "DynamoDBRequestStore"}
```

Comment:
Why keeping the **Mongo** request store?

Reply:
The intention is to merge this upstream and, as far as I know, ECMWF intends to continue using mongodb